src/BinaryGapOrzech.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by Łukasz on 2017-04-09.
+ */
+public class BinaryGapOrzech {
+    public int solutionOrzech(int N){
+        String nat=Integer.toBinaryString(N);
+        List<String> myList = new ArrayList<String>(Arrays.asList(nat.split("")));
```

Comment:
myList to bardzo slaba nazwa dla zmiennej, przy okazji masz stringa a zamieniasz go na tablice a pozniej na liste, nie wiem po co, skorzystaj ze stringa, mozesz po nim przeleciec jak po liscie, uzyj metod lenth i charAt